R/tm_g_distribution.R

```diff
@@ -118,7 +118,8 @@ tm_g_distribution <- function(label = "Distribution Module",
                               plot_height = c(600, 200, 2000),
                               plot_width = NULL,
                               pre_output = NULL,
-                              post_output = NULL) {
+                              post_output = NULL,
+                              card_function) {
```

Comment:
```suggestion
                              card_function = tm_g_distribution_card_function) {
```

I would expose this so that it's visible which function is used. However that would require exposing N reporting cards for N modules